src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/crafting/AnvilListener.java

```diff
@@ -15,8 +15,8 @@
 import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
 
 /**
- * This {@link Listener} prevents any {@link SlimefunItem} from being used in a
- * {@link BrewingStand}.
+ * This {@link Listener} prevents any {@link SlimefunItem} from being used in an
+ * anvil.
```

Comment:
link pls

Reply:
No link to Anvil, just to AnvilInventory (at least that I saw). All other Listener classes have the machine just typed out like that too so I went with it

Reply:
Well the brewing stand has a link though, so I would argue in favour of uniformity to remove the link from the brewing stand listener then